### ISSUE TITLE
Image overlays in Essay Question type are not interactive

### ISSUE THREAD

**Issue by yannisbarlas** [Coko-Foundation/wax-prosemirror#17]
> **Migrated from GitLab**
> **Original Author:** Vukile Langa (`@vukilelanga`)
> **Original Link:** https://gitlab.coko.foundation/wax/wax-prosemirror/-/issues/110

## Expected behaviour

A user should be able to edit an image's alt text and optional long description in an essay question.

## Current behaviour

When clicking on the alt text/long description overlay, the overlay is immediately hidden and does not allow the user to edit the values.

## Steps to reproduce

* Create a new question type of `Essay`
* Add an image to either the essay question, or the essay answer below
* Click on the image. The alt text overlay should appear
* Click on the overlay to edit the alt text. The overlay should disappear

This can be seen on the `Widget Example` in the [wax demo site](https://waxeditor.cloud68.co).

## Environment

This occurs in all major browsers and operating systems.

## Possible solution

A refactor of the `EssayService`, to remove the `EssayQuestionContainerComponent`. The goal is to make the Essay questions more in line with functionality seen in Multiple Choice questions.
